cosmos-init/plugins/packages/cosmosc2-tool-tablemanager/src/tools/TableManager/TableManager.vue

```diff
@@ -286,16 +351,16 @@ export default {
             {
               divider: true,
             },
-            {
-              label: 'Upload',
-              icon: 'mdi-cloud-upload',
-              command: () => {
-                this.upload()
-              },
-            },
-            {
-              divider: true,
-            },
+            // {
+            //   label: 'Load Binary',
+            //   icon: 'mdi-cloud-upload',
+            //   command: () => {
+            //     this.loadBinary()
+            //   },
+            // },
+            // {
+            //   divider: true,
+            // },
```

Comment:
What is going on here?

Reply:
I want the ability to save a binary file from the local filesystem into Minio. Didn't get it to work and didn't want to mess with it with this ticket.